lib/mustache/generator.rb

```diff
@@ -152,6 +152,34 @@ def on_partial(name, offset, indentation)
       ev("ctx.partial(#{name.to_sym.inspect}, #{indentation.inspect})")
     end
 
+    # Fired when the compiler finds a inheritance. We want to return code
+    # which stores the block variables in this template and  
+    # calls the parent at runtime.
+    def on_parent(name, offset, content, raw, delims)
+      result = ""  
+      parent_name = name[2].first  
+      content.each do |x| 
+        if x != :multi && x[0] === :mustache && x[1] === :blockvar then
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Do not use `then` for multi-line `if`.](https://www.codacy.com/app/rokusu/mustache/file/1131006119/issues/source?bid=2903613&fileBranchId=2904056#l162)